src/cli.js

```diff
@@ -142,6 +142,122 @@ function splitIssueBodyIntoSections(body) {
 }
 
 
+/**
+ * Return a program handler for the "monitor/ignore" actions. Both actions
+ * are extremely similar, this avoids code duplication.
+ */
+function getMonitorOrIgnoreHandler(action) {
+  if (action !== 'monitor' && action !== 'ignore') {
+    throw new Error(`Unknown action ${action}`);
+  }
+  return async function (what, comment, options) {
+    // Set implicit options
+    if (options.pr) {
+      options.commit = true;
+    }
+
+    // Function used to report on progress
+    const logProgress = options.quiet ? function () {} : console.warn;
+
+    // Retrieve the actual parameters from GitHub if "what" is an issue number
+    let issueNumber = null;
+    if (what.match(/^\d+$/)) {
+      logProgress(`Retrieve issue info from GitHub...`);
+      issueNumber = what;
+      let issueStr = null;
+      try {
+        issueStr = execSync(`gh issue view ${what} --json body,state,title,comments`, execParams);
+      }
+      catch (err) {
+        console.log(`Could not retrieve issue ${issueNumber}.`);
+        process.exit(1);
+      }
+      const issue = JSON.parse(issueStr);
+      const sections = splitIssueBodyIntoSections(issue.body);
+      const urlSection = sections.find(section => section.title === 'URL');
+      if (!urlSection) {
+        console.log(`Issue #${issueNumber} does not follow the expected structure.`);
```

Comment:
That would be a valuable improvement, but the command does not report anything on the issue itself for a reason: that would not work well (as-is) with the job that checks spec suggestion issues, should the issue description be further edited, because the job currently overrides the last comment from the bot (expecting that comment to be the previous analysis).

The problem is that the `gh` command does not allow to edit a specific issue comment. It only allows to edit the *last* comment made by an author, or to create a new comment. We'd need to change the logic somehow, for example using the GraphQL endpoint directly to edit the comment instead of the `gh` CLI, or making these reports under another name. I don't mind the occasional job failure in the meantime ;)